### ISSUE TITLE
About implementation details

### ISSUE THREAD

**Issue by Katono5** [megvii-research/BasesHomo#7]
Question1:
Based on the requirements said in README, you guys use pytorch1.1.0 to train the model, and based on the paper you guys use 2080ti gpu. But when I tried to implement baseshomo on 2080ti, it would return an error since pytorch1.1.0 use cuda9.0 which 2080ti don't support, so I really wonder how you guys deal with the problem.

Question2:
The paper said the learning rate would reduce 20% every epoch but based on the code given, I didn't change any part of the code and the learning rate is fixed at 0.0001
![image](https://github.com/megvii-research/BasesHomo/assets/61302913/d5438783-1480-47e5-86ba-e7a41f040997)

Im really interested in your work and hope you could generously give me a hand following your work

**Comment by lhaippp** [comment]
Hi, 
1. I think it doesn't matter what cuda version you use, have a try with any matching cuda version.
2. The code seems to be correct, the problem is the  [Floating point digits](https://github.com/megvii-research/BasesHomo/blob/af2c5527e17ee1d23abd8b60673c0940e586385c/common/manager.py#L104C46-L104C46)
```
print_str = "{} Epoch: {:4d}, lr={:.4f} ".format(exp_name, self.epoch, self.scheduler.get_lr()[0])
# change it to
print_str = "{} Epoch: {:4d}, lr={:.5f} ".format(exp_name, self.epoch, self.scheduler.get_lr()[0])
```

**Comment by Katono5** [comment]
> Hi,
> 
> 1. I think it doesn't matter what cuda version you use, have a try with any matching cuda version.
> 2. The code seems to be correct, the problem is the  [Floating point digits](https://github.com/megvii-research/BasesHomo/blob/af2c5527e17ee1d23abd8b60673c0940e586385c/common/manager.py#L104C46-L104C46)
> 
> ```
> print_str = "{} Epoch: {:4d}, lr={:.4f} ".format(exp_name, self.epoch, self.scheduler.get_lr()[0])
> # change it to
> print_str = "{} Epoch: {:4d}, lr={:.5f} ".format(exp_name, self.epoch, self.scheduler.get_lr()[0])
> ```

Thanks for responding!
Just as a reminder, accroding to [https://github.com/megvii-research/BasesHomo/issues/4#issue-1066718730](url) , the code now is still remain unchanged lol, I've tried to use pytorch1.11.0 to train and the code runs well though cannot reproduce the result, but if I follow the requirement strictly, the code would return an error.

I hope this would help others who try to reproduce your work.
